build.gradle

```diff
@@ -97,6 +97,8 @@ ext {
     awsCommand = "aws"
 
     sdkZip = file("${buildDir}/java-sdk-${version}.zip")
+
+    swaggerVersion = "1.5.16"
```

Comment:
Okay, so I've tried that, and it has a bunch of rough edges:

- either I'm missing something, or it doesn't seem to affect `buildScript` dependencies, which is where this problem was introduced
- it doesn't inherit settings in subprojects. You either have to configure it everywhere with `subprojects { }`, or manually include it in each project. So you still need the equivalent of a `${swaggerVersion}` substitution to wire those together.

I've already sunk almost an hour on this; I now no longer think it makes sense to try to use this plugin for this particular issue. We can add it once we depend on the AWS SDK in more places.